OmniBLE/PumpManager/PodCommsSession.swift

```diff
@@ -99,6 +100,8 @@ extension PodCommsError: LocalizedError {
             return LocalizedString("Too many pods found", comment: "Error message for PodCommsError.tooManyPodsFound")
         case .setupNotComplete:
             return LocalizedString("Pod setup is not complete", comment: "Error description when pod setup is not complete")
+        case .possibleBluetoothIssue:
+            return LocalizedString("Possible Bluetooth issue", comment: "Error description for possible bluetooth issue")
```

Comment:
I think we know this *is* a bluetooth communication issue, right?  I think the set of errors you're wrapping here cover everything from the pod being out of range, to the kind that might need a bluetoothd reset. The former is generally more likely, and easier to resolve.

Reply:
Use of this new error is currently limited to low level (hopefully bluetooth only) commsErrors during pairing. Currently there are PodCommsErrors that can be returned in different error cases that try to provide more helpful errors & recovery suggestions for other particular cases that are detected and returned (e.g., .noResponse, .podNotConnected, .rssiTooLow, .noPodsFound, .tooManyPodsFound, etc) which won't be affected. In (nearly?) every case of a commError error during pairing that I've encountered with an actual pod where a retry didn't work, I've needed to reset the iPhone bluetooth (or reboot the phone) to be able to recover and no other PodCommErrors have this recovery suggestion. Hopefully use this new error will be limited to specific cases where this suggested recoverySuggestion will be valid (and is the only real recovery possible), but it should still work for other comms issue cases that aren't already being being detected.

Thinking about this now, perhaps an alternative approach might be to change the current nil recoverySuggestion for PodCommsError.commsError to "Toggle Bluetooth off and then on in iPhone Settings; then try again." This approach does have some nice properties, but it would change other places in Loop where this recoverSuggestion might be optionally displayed (potentially in some negative way).